hera_cal/frf.py

```diff
@@ -26,22 +28,21 @@ def timeavg_waterfall(data, Navg, flags=None, nsamples=None, rephase=False, lsts
     weighted by a boolean flag array (flags) and also optionally by an Nsample array (nsample),
     such that, for a single frequency channel, the time average is constructed as
 
-    avg_data = sum( data * flag_wgt * nsample ) / sum( flag_wgt * nsample )
+    avg_data = sum( data * flag_wgt * nsample**2 ) / sum( flag_wgt * nsample**2 )
 
     where flag_wgt is constructed as (~flags).astype(np.float).
 
     Additionally, one can rephase each integration in the averaging window to the LST of the
     window-center before taking their average. This assumes the
     input data are drift-scan phased. See hera_cal.utils.lst_rephase
     for details on the rephasing algorithm. By feeding an nsample array,
-    one can also construct the averaged nsample for each averaging window.
+    the averaged nsample for each averaging window is computed and returned.
 
     Parameters
     ----------
     data : ndarray
         2D complex ndarray of complex visibility with shape=(Ntimes, Nfreqs)
-        The rows of data are assumed to be ordered chronologically, in either
-        asending or descending order.
+        The rows of data are assumed to be ordered chronologically.
```

Comment:
Is equal spacing in time assumed?

Reply:
i don't think this is assumed, but also I don't think there is a scenario in our normal day-to-day where that wouldn't be the case..